project/profiles/capacity4more/modules/c4m/mail/c4m_mail/messages/user/user_mail_status_activated_body.html

```diff
@@ -5,7 +5,7 @@
 <p><br>
   You’re now a member of [site:name]! You can join Groups, upload posts, add comments, create projects and launch your own communities of practice.
   <br>
-  Learn more about what capacity4dev can do for you in our <a href="[site:url]/help-guidance">help section</a>.
+  Learn more about what capacity4dev can do for you in our <a href="[site:url]/help-guidance/help-guidance">help section</a>.
```

Comment:
The _href_ should be:
`[site:url]help-guidance/help-guidance`
otherwise the link will contain double `/` ...